### ISSUE TITLE
Wrong Naming for Dimension Id in rftoolsdim-1.12-5.04 ?

### ISSUE THREAD

**Issue by PapaSteeni** [McJty/RFTools#1405]
I'm running the FTB Unstable Pack on a dedicated server. Dimension are created, dialer works, analyzer shows green light, but actually teleporting to a dimension crashes the server with following logs:

Crashlog:[ https://pastebin.com/RfahpYcb]( https://pastebin.com/RfahpYcb)
Minecraft Log; [https://pastebin.com/EhRNxYz1](https://pastebin.com/EhRNxYz1)

Tells me that I'm trying to teleport to a non-existing DIM. Which is true as DIM4 does not exist, RFtools created a folder named "RFTOOLS4" to store the dimension data in. Maybe I'm not understanding something here, but it seems RFtools is creating the dimension data in a wrong folder, because even using the /rftdim safedel 4 command, answers me that the dimension got deleted, the player lost in that dimension gets relocated to Worldspawn, but the folder "RFTOOLS4" does not get deleted. And yes, I've enabled the option to delete dimension data with the /rtfdim safedel command.
I created several random dimensions using the digit dims to alter the Id, the problem persists...whatever the Id is.

**Comment by McJty** [comment]
Does this happen with every dimension or just with dimension 4? The folder name is right. That's how it is supposed to be

**Comment by PapaSteeni** [comment]
Like I said....I created several random dimensions using the digit dimlet to alter the dimension id....happens always, so far id3 to id9 tested...
Update: Using the /rfdim safedel command sometimes does delete the corresponding dimension folder and its data, the crash trying to enter the dimension still occurs everytime though...I'll try some higher ids now.

**Comment by McJty** [comment]
Can You try updating to latest and see if the problem remains?

**Comment by PapaSteeni** [comment]
I'll brake the pack for my users that way, but I'll do a quick test right away...

**Comment by PapaSteeni** [comment]
Updated rftools and rfdim to latest available on curseforge, ctrl was still on same version.
Still server crash on trying to reach the dimension:
[Crashlog](https://pastebin.com/vkCLrcJJ)

**Comment by McJty** [comment]
Yes that dimension is probably broken  the question is if new dimensions are ok
